### PR TITLE
Updated compilation instructions for Visual Studio

### PR DESCRIPTION
**Pull request by CrunchBite**
This updates the instructions for compiling with Visual Studio 2019, adds the appropriate CMake Settings and sets the build target to Windows 7 when using MSVC.

### CODE REVIEW

**Review by codedwrench — COMMENTED** [review]
-

File: CMakeSettings.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿{
```
Comment (codedwrench) [inline CMakeSettings.json:1]: I think this file needs to go, it's setup specific

File: main.cpp [hunk 2]
```diff
@@ -23,6 +23,11 @@
 #include "Includes/WirelessPromiscuousDevice.h"
 #include "Includes/XLinkKaiConnection.h"
 
+// Set target to Windows 7 for MSVC
+#if defined(MSVC) && (defined(_WIN32) || defined(_WIN64))
```
Comment (codedwrench) [inline main.cpp:27]: This should be put into the cmake, not the code

File: CMakeSettings.json (hunk 1 above)
Comment (codedwrench) [reply CMakeSettings.json:1]: You could make it empty strings sure, or put them on ../ like i do with linux, but they are cache variables, so after first generation you should be able to just set them, that's why they're predefined so they show up in the mneu.

Comment (CrunchBite) [reply CMakeSettings.json:1]: The paths are now generic `../`

File: main.cpp (hunk 2 above)
Comment (CrunchBite) [reply main.cpp]: Moved to CMakeLists.txt

**Pull request merged**